test/unit/cookie.test.js

```diff
@@ -194,4 +194,26 @@ describe('Cookie', function () {
             })).valueOf()).to.eql('this is a cookie value');
         });
     });
+
+    describe('unparse', function () {
+        it('should return empty string if not array of cookies', function () {
```

Comment:
The `it` block name does not specify what exactly is being tested here.
```suggestion
        it('should return empty string if called with invalid cookies array', function () {
```